src/components/ConfirmDialog.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <v-dialog @input="change" value="true" max-width="500px">
+    <v-card class="text-xs-center pa-3">
+      <v-icon large color="warning" class="warning-icon">warning</v-icon>
+      <v-card-text class="headline">{{ message }}</v-card-text>
+      <v-card-actions>
+        <v-spacer></v-spacer>
+        <v-btn color="primary" flat="flat" @click="choose(false)">
```

Comment:
The false response could maybe be a different color

Reply:
Agree, but that's the way we usually have Cancel button (following Material Design)